### PR TITLE
/announce channel-status|volunteer-status [preamble]

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: src/slack_interaction_handler.ts [hunk 1]
```diff
@@ -666,6 +666,164 @@ export async function handleCommandNeedsAttention(
...
+    );
+    return;
+  }
+
+  let preamble = '';
+  const args = text.split(' ');
+  if (!args) {
+    return;
+  }
+  const what = args.shift();
```
Comment (tomerovadia) [inline src/slack_interaction_handler.ts]: Can you rename the `what` variable to something more descriptive?

File: src/slack_interaction_handler.ts [hunk 2]
```diff
@@ -666,6 +666,164 @@ export async function handleCommandNeedsAttention(
...
+        responseUrl,
+        `Sent announcements to ${channels.length} channels.`
+      );
+      return;
+    }
+    case 'volunteer-status': {
+      if (preamble) {
+        preamble += '\n\n';
+      }
+      const vstats = await DbApiUtil.getThreadsNeedingAttentionByVolunteer();
```
Comment (tomerovadia) [inline src/slack_interaction_handler.ts]: nit: can you rename vstats to volunteerStats?

File: src/slack_interaction_handler.ts [hunk 3]
```diff
@@ -666,6 +666,164 @@ export async function handleCommandNeedsAttention(
...
+      for (const v of vstats) {
+        const ulines = await getNeedsAttentionList(v.volunteerSlackUserId);
```
Comment (tomerovadia) [inline src/slack_interaction_handler.ts:802]: Can you rename ulines to something more descriptive?

File: external_config/slack.md [hunk 4]
```diff
@@ -183,3 +183,10 @@ This should be added as a **global** shortcut.
 - **Request URL:** base URL + /slack-command
 - **Short Description:** List voters needing your attention
 - **Usage Hint:** (None)
+
+#### Announce
+
+- **Command:** /announce
```
Comment (tomerovadia) [inline external_config/slack.md]: Can we rename /announce to something more specific?

**Review by tomerovadia — CHANGES_REQUESTED** [review]
Added some minor comments. Happy to push to staging tomorrow (and can include Eli's changes as well, if you like them).

File: external_config/slack.md (hunk 4 above)
Comment (liewegas) [reply external_config/slack.md]: how about `/broadcast`?

**Review by tomerovadia — APPROVED** [review]

**Pull request merged**